src/views/Transfer/Transfer.html

```diff
@@ -38,22 +38,31 @@
             <div style="width: 100%; display: flex;">
               <span class="text-capitalize">{{ data.item.name }}</span>
               <v-spacer></v-spacer>
-              {{ data.item.balances[0].balance | formatBalance }} {{data.item.balances[0].asset_code }}
+              {{data.item.balances.find(x=>x.asset_code == selectedCurrency).balance | formatBalance}}
```

Comment:
Should be moved to computed or method, too much logic in templates can be confusing